src/applications/claims-status/components/AddFilesForm.jsx

```diff
@@ -301,21 +316,18 @@ class AddFilesForm extends React.Component {
             Cancel
           </Link>
         </div>
-        <Modal
-          onClose={() => true}
-          visible={this.props.uploading}
-          hideCloseButton
-          cssClass=""
+        <VaModal
           id="upload-status"
-          contents={
-            <UploadStatus
-              progress={this.props.progress}
-              files={this.props.files.length}
-              onCancel={this.props.onCancel}
-            />
-          }
-        />
-      </div>
+          onCloseEvent={() => true}
+          visible={Boolean(this.props.uploading)}
```

Comment:
The `Modal` component had a default prop of `false` for the `visible` prop. The web component doesn't so visible is set to undefined if the `AddFilesForm` component doesn't have the `visible` prop set